app/assets/stylesheets/research/submission-cancel-confirmation.scss

```
@@ -0,0 +1,58 @@
+@import 'variables';
+
+body.namespace-research {
```

Can we scope this to the experiment-solution page pls? 

```suggestion
body.namespace-research.controller-experiment_solutions.action-show {
```